src/hero/_base.scss

```diff
@@ -7,6 +7,7 @@
 .hero {
   display: flex;
   flex-direction: column;
+  align-items: center;
   justify-content: center;
   min-height: 300px;
```

Comment:
I couldn't find a nice viewport unit, so I'm just going to use a modular scale value for now until I can work with design a bit more on that.